rl_loop/train_and_validate.py

```diff
@@ -53,10 +56,14 @@ def train():
     new_model_name = shipname.generate(new_model_num)
     print("New model will be {}".format(new_model_name))
     save_file = os.path.join(fsdb.models_dir(), new_model_name)
+    #freeze_tpu requires the work_dir flag to be set
+    if not FLAGS.work_dir:
```

Comment:
i'm confused by this - this wrapper process shouldn't be directly executing any code; it should be subprocessing out as `python3 freeze_graph.py...`. Passing in flags here is incorrect.

Reply:
ok, let me rework it.

Reply:
ok i've refactored this to call out to freeze_graph; PTAL?